README.md

```diff
@@ -73,7 +89,7 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
 ```
 
-### 3. Provider-Specific Configurations
+### 4. Provider-Specific Configurations
```

Comment:
🔧 Good to include provider-specific configurations! This helps users customize their setup.